www/docs/schemas/adapters.md

```diff
@@ -7,6 +7,24 @@ An **Adapter** in NextAuth.js connects your application to whatever database or
 
 You do not need to specify an Adapter explicitly unless you want to use advanced options such as custom models or schemas, if you want to use the Prisma Adapter instead of the default TypeORM Adapter, or if you are creating a custom Adapter to connect to a database that is not one of the supported databases.
 
+We have recently begun moving the adapter code out of the core of `nextauthjs/next-auth` and into their own repository under [`nextauthjs/adapters`](https://github.com/nextauthjs/adapters).
```

Comment:
I think the migration is done, so I would rephrase this so we won't need to revisit this sentence later on. Something like adapters are separate from core and they can be found under...